syntax = "proto3";

package persister;

message LogEntry
{
    uint64 term = 1;
    bytes command = 2;
}

message LogEntryVector
{
    repeated LogEntry entries = 1;
}